### DISCUSSION TITLE
关于对逐火英桀第二位——“爱莉希雅”称号“真我”的宣传报告

### DISCUSSION THREAD

**Discussion by azb-666** [share121/inter-knot#3253] [其他]
![爱莉希雅](https://github.com/user-attachments/assets/47feb68b-2ce4-4400-a4ec-0e1fd8a815b0)

如头图所示，想你了，牢爱

**Comment by shawnho1029** [comment]
真的加了我去STEAM給他打個負評(DOGE
![1658316c69dfb1fe973954e29bc98f9b1486834703](https://github.com/user-attachments/assets/0a2c99ee-9148-4859-94b4-e1a19e396cb1)

**Comment by s-j-y-m** [comment]
这不得来个觉醒技变人律（

**Comment by azb-666** [comment]
人律有哦



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月20日 11:45 |
| 收件人 | ***@***.***> |
| 抄送至 | EAD-MS (External Affairs Department of the Moth ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 关于对逐火英桀第二位——“爱莉希雅”称号“真我”的宣传报告 (Discussion #3253) |

这不得来个觉醒技变人律（

**Comment by HyperMuteki0347** [comment]
好蒸的卡面<img src="https://img-static.mihoyo.com/communityweb/upload/1af903e950e4be03cfa0c7e9935605c1.png" alt="阿贝多-怎么了" width="75" >
